lib/store/settings_store.dart

```diff
@@ -789,12 +790,9 @@ abstract class SettingsStoreBase with Store {
     final exchangeStatus = ExchangeApiMode.deserialize(
         raw: sharedPreferences.getInt(PreferencesKey.exchangeStatusKey) ??
             ExchangeApiMode.enabled.raw);
-    final legacyTheme = (sharedPreferences.getBool(PreferencesKey.isDarkThemeLegacy) ?? false)
-        ? ThemeType.dark.index
-        : ThemeType.bright.index;
     final savedTheme = initialTheme ??
         ThemeList.deserialize(
-            raw: sharedPreferences.getInt(PreferencesKey.currentTheme) ?? legacyTheme);
+            raw: sharedPreferences.getInt(PreferencesKey.currentTheme) ?? (isMoneroOnly ? 3 : 2));
```

Comment:
use theme index better, for readability and maintainability if we change their index at any given time
ex.
`? ThemeType.bright.index : ThemeType.bright.index`

Reply:
I can't reference the monero orange theme with the ThemeType index. Seems to be for just a type of theme. not one of the specific new themes.

Reply:
Managed to do it in a cleaner way by grabbing from ThemeList.[theme].raw instead :smile: